### ISSUE TITLE
Proposed Removal of Additional Resources from Ruby Tic-Tac-Toe Project

### ISSUE THREAD

**Issue by msespos** [TheOdinProject/curriculum#23171]
I'm making this issue to propose that the three Additional Resources in the Ruby Tic-Tac-Toe Project be removed.

The first resource is already labeled as "not directly helpful", and it really seems irrelevant to the project at hand.
The second resource is a walkthrough of coding a Tic-Tac-Toe game, which also includes testing, so that even if the user does attempt to code through the walkthrough (which is in itself problematic), they will also face having to learn tests well before they appear in the Odin curriculum.
The third resource is simply the code for a full solution of Tic-Tac-Toe, which is extremely problematic in my opinion.

Given that the general approach at Odin is "don't look at the solutions until you've finished the project," providing solutions before the user has even started as Additional Resources is at best contradictory and/or misleading, and at worse a blueprint for a student to not create their own project. I realize that most students will not fall into this trap, but I still think the argument that those resources are misleading is enough for them to be dropped.

tl;dr the first resource is superfluous, and the second and third provide solutions and thus should not be made available at the start of the project.

**Comment by msespos** [comment]
Just want to say I know there could very well be good arguments for keeping any or all of these resources. These are just my arguments for removing them.

**Comment by zachmmeyer** [comment]
I have had and heard this exact discussion before. I do not believe you are alone in wanting the additional resources to be removed for this project. I am personally fully for it. I do think we need a second opinion.

**Comment by rlmoser99** [comment]
I am for removing these from the curriculum. @msespos would you be willing to make the PR?

**Comment by msespos** [comment]
I can definitely make the PR. I'll do it tomorrow. Thanks!
